NAMESPACE

```diff
@@ -1,6 +1,11 @@
 # Generated by roxygen2: do not edit by hand
 
+export(dxdt)
```

Comment:
what does this do? Can we give name such as `calculatedxdt`. If this does not have to be exported, then this does not matter and better to keep the name as is. If this must be expoerted, the name should be more descriptive.